core-bundle/src/Twig/Extension/ContaoExtension.php

```diff
@@ -69,24 +69,19 @@ public function __construct(
         private readonly ContaoVariable $contaoVariable,
     ) {
         $contaoEscaper = new ContaoEscaper();
-        $escaperExtension = $environment->getExtension(EscaperExtension::class);
 
-        // Forward compatibility with twig/twig >=3.10.0
-        if (method_exists($escaperExtension, 'setEnvironment')) {
-            $escaperExtension->setEnvironment($environment);
-        }
-
-        $escaperExtension->setEscaper('contao_html', $contaoEscaper->escapeHtml(...));
-        $escaperExtension->setEscaper('contao_html_attr', $contaoEscaper->escapeHtmlAttr(...));
+        $runtime = $this->environment->getRuntime(EscaperRuntime::class);
```

Comment:
Can we name the variable `$escaperRuntime` everywhere, so it is clear that we do not mean any of the other runtimes? ☺️

Reply:
Changed in 6f7ccc2f4c60b862ab65cb0a8ff17ffc4432060c.